apps/webapp/src/app/(Magic-Link)/magic-link/page.tsx

```
@@ -0,0 +1,614 @@
+      return [];
+    });
+  }
+
+  const onCloseBasicAuthDialog = (dialogState: boolean) => {
+    setOpenBasicAuthDialog(dialogState);
+    reset2();
+  };
+
+  const onBasicAuthSubmit = (values: IBasicAuthFormData) => {
+    onCloseBasicAuthDialog(false);
+    setLoading({ status: true, provider: selectedProvider?.provider! });
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)